### PR TITLE
Add footer on every page

### PR DESCRIPTION
**Pull request by Zarathustra4**
image

### CODE REVIEW

File: static/styles/footer.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+footer {
+    margin-top: 70px;
+    max-width: 100%;
+    display: flex;
+    background-color: rgb(5, 5, 62);
+    padding: 50px 20px;
+    color: white;
```
Comment (mentosf) [inline static/styles/footer.css:7]: change color to brown

File: static/styles/footer.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+footer {
+    margin-top: 70px;
```
Comment (mentosf) [inline static/styles/footer.css:2]: change margin-top to 80px

File: index.html [hunk 3]
```diff
@@ -148,7 +148,24 @@ <h2>Event title</h2>
...
+            <h2 class="footer-text">Bill Gates</h2>
+        </div>
+        <div class="right">
+            <ul>
+                <li>billgates@gmail.com</li>
+                <li>+380344238392</li>
+                <li>https://linkedin.com/billgates</li>
+            </ul>
+            <ul>
+                <li><a href="" class="">link 1</a></li>
```
Comment (mentosf) [inline index.html:163]: remove empty class

**Review by mentosf — CHANGES_REQUESTED** [review]
General code is ok. But its still some issues

File: static/styles/footer.css (hunk 1 above)
Comment (Zarathustra4) [reply static/styles/footer.css]: fixed

File: static/styles/footer.css (hunk 2 above)
Comment (Zarathustra4) [reply static/styles/footer.css]: fixed

File: index.html (hunk 3 above)
Comment (Zarathustra4) [reply index.html]: fixed

**Pull request merged**